.bitmap

```diff
@@ -4646,6 +4646,54 @@
         "exported": true,
         "lanes": []
     },
+    "sidebar/ui/link-section": {
+        "files": [
+            {
+                "name": "index.ts",
+                "relativePath": "index.ts",
+                "test": false
+            },
+            {
+                "name": "link-section.module.scss",
+                "relativePath": "link-section.module.scss",
+                "test": false
+            },
+            {
+                "name": "link-section.tsx",
+                "relativePath": "link-section.tsx",
+                "test": false
+            }
+        ],
+        "mainFile": "index.ts",
+        "rootDir": "scopes/ui-foundation/link-section",
+        "origin": "AUTHORED",
+        "exported": false,
+        "lanes": []
+    },
+     "ui/sidebar-link": {
```

Comment:
please rename sidebar-link and sidebar/ui/link-section.
Try to make a 'menu' oriented design that we could reused across the system.

You can also split between ui-foundation, design, and sidebar scopes

```
manu
  item
  item
  item
  section
    item
    separator
    item
```

Reply:
done